pkg/ansible/run.go

```diff
@@ -148,12 +148,25 @@ func Run(flags *aoflags.AnsibleOperatorFlags) error {
 	}
```

Comment:
Missing CHANGELOG entry. WDYT of something as:

| Add support to generate ServiceMonitor resources to export metrics on Ansible based operators.

Reply:
Solved.